dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -81,6 +82,19 @@ static ArgumentParser getArgumentParser() {
                 .type(Boolean.class)
                 .setDefault(false)
                 .help("Only update image tag store. Skip creating PRs");
+        parser.addArgument("-u", "--" + Constants.USE_RATE_LIMITING)
```

Comment:
Why do we need this flag? If rate limit is not set, then this would be false.

Reply:
we have kept this flag to provide a default rate limit. If only this flag value is true, then the default rate Limit will be set.

Reply:
The default rate limit should be none - I still don't think this is necessary